### ISSUE TITLE
Any suggestion for adding a track for log2 ChIP/Input (bigwig file）

### ISSUE THREAD

**Issue by wzylin** [jokergoo/circlize#254]
Hi Dr. Gu，
I have generated bigwig files with the log2 ratio of ChIP/Input .
Could you give me some suggestion for adding a track of data from the bigwig file?
Thank you!

**Comment by jokergoo** [comment]
Hi the question is too general that sorry I cannot help you...

**Comment by wzylin** [comment]
> Hi the question is too general that sorry I cannot help you...

I have figured out how to plot the bw file. I converted the bw to bedgraph.

May I asked another question about genomicDensity?

`circos.genomicDensity(bed, col = c("#FF000080"), count_by = "number", window.size = 1e5, track.height = 0.15, force.ylim = TRUE)`
Above is the code I used, but is it possible to show the genomicDensity as heatmap instead of peaks?
like this:
![image](https://user-images.githubusercontent.com/23481140/117931121-625d8e80-b331-11eb-89cf-9c29ad2709c0.png)

Thank you!
